participants/jhanvi12.md

```diff
@@ -1,5 +1,46 @@
-Name: Jhanvi uday
-GitHub username: Jhanvi12
-University: IGDTUW
-Year: 1
-Branch:Mtech -IT(ISM)
+
+>#  AI yields new Antibiotic
+>  **A deep-learning model identifies a powerful new drug that can kill many species of antibiotic-resistant bacteria**
+> [Science News]([https://www.sciencedaily.com/releases/2020/02/200220141748.htm](https://www.sciencedaily.com/releases/2020/02/200220141748.htm))
+
+> *Date* &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;     :  &nbsp;&nbsp; &nbsp;&nbsp;         `February 20, 2020`
+
+> *Source*  &nbsp;&nbsp;  : &nbsp; &nbsp; &nbsp;   ` Massachusetts Institute of Technology `
+
+> *Summary*:   &nbsp; &nbsp; Using a machine-learning algorithm, researchers      have identified a powerful new antibiotic compound. In laboratory tests, the drug killed many of the world's most problematic disease-causing bacteria, including some strains that are resistant to all known antibiotics. It also cleared infections in two different mouse models.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<u>***Full Story***</u>
+
+"The computer model, which can screen more than a hundred million chemical compounds in a matter of days, is designed to pick out potential antibiotics that kill bacteria using different mechanisms than those of existing drugs."
+
+"We wanted to develop a platform that would allow us to harness the power of artificial intelligence to usher in a new age of antibiotic drug discovery," says James Collins, the Termeer Professor of Medical Engineering and Science in MIT's Institute for Medical Engineering and Science (IMES) and Department of Biological Engineering. "Our approach revealed this amazing molecule which is arguably one of the more powerful antibiotics that has been discovered."
+<a href="https://www.youtube.com/watch?v=2GTFB8Zs_Q4&t=5s"><img src="http://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg" 
```

Comment:
If this was a thumbnail then it doesn't actually work as you can see [here](https://github.com/ACM-IGDTUW/Open-Source-Challenge/blob/690282bc3f4726606bd8906c3895f370f5369923/participants/jhanvi12.md)

Reply:
@lisa761 ,it was working on stack edit , thereby I have implemented it